rotkehlchen/api/v1/resources.py

```diff
@@ -1455,23 +1511,22 @@ def make_put_schema(self) -> EvmAccountsPutSchema:
         )
 
     @require_loggedin_user()
-    @resource_parser.use_kwargs(make_put_schema, location='json_and_query')
-    def put(
-            self,
-            accounts: list[dict[str, Any]],
-            async_query: bool,
-    ) -> Response:
+    @resource_parser.use_args(
+        make_put_schema,
+        location='json_and_query',
+        allow_async_validation=True,
+        as_kwargs=True,
```

Comment:
why is as kwargss needed? Add a comment. Also short comment next to `allow_async_validation=True` on what it does.

Reply:
you need to add a comment here explaining why as I said in original request.